controllers/servers/host_definer/kubernetes_manager/manager.py

```diff
@@ -193,61 +211,63 @@ def _get_status_manifest(self, host_definition_phase):
 
         return status
 
-    def _get_event_for_host_definition(self, host_definition, message, action, message_type):
+    def _generate_k8s_event_for_host_definition(self, host_definition_info, message, action, message_type):
```

Comment:
do we have k8s_event generation for another purpose than host_definition

Reply:
no

Reply:
I changed it to `_generate_k8s_event`